source/scripts/app.js

```diff
@@ -932,6 +949,18 @@ function bindExplorePageLearnMore() {
     let cardShadow = recipeCards[i].shadowRoot;
     let button = cardShadow.getElementById("recipe-info-button");
     button.addEventListener("click", redirectToRecipe);
+
+    /* jshint ignore:start */
+    let addButton = cardShadow.getElementById("recipe-card-action-button");
+    addButton.addEventListener("click", () => {
+      let recipeId = cardShadow.getElementById("recipe-id");
+      globalRecipeId = recipeId.textContent;
```

Comment:
You can avoid using `globalRecipeId` if you create a setter and getter inside the `notification-select-cookbook` custom element class to set and get a recipe ID

Reply:
Added the setter and getter and removed global variable